src/app/app.component.css

```diff
@@ -1,8 +1,21 @@
+@import url('@scania/tegel/dist/tegel/tegel.css');
+@import url('https://cdn.digitaldesign.scania.com/icons/webfont/css/sdds-icons.css');
```

Comment:
Is this needed? 
Our icons come as a web-component now, so no need to import anything related to SDDS code.
Please double-check.